sdk/chaos/Azure.ResourceManager.Chaos/tests/Scenario/ExperimentTests.cs

```diff
@@ -70,23 +70,26 @@ public async Task Delete()
             var deleteResponse = await experimentResourceResponse.Value.DeleteAsync(WaitUntil.Completed).ConfigureAwait(false);
             Assert.AreEqual(200, deleteResponse.GetRawResponse().Status);
 
+            await Task.Delay(2000);
```

Comment:
Consider to use [RecordedTestBase.Delay](https://github.com/Azure/azure-sdk-for-net/blob/7eba637ebc9929a645abe3ca08b821aa349a5d52/sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs#L504)